### PR TITLE
Acknowledging the user with future outrages : Worked on remote repo and modified it.

### PR DESCRIPTION
**Pull request by SUMEKAGARWAL**

### CODE REVIEW

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 Just have a look on this.

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 @CodesbyUnnati May I have the confirmation please!

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 @CodesbyUnnati Just have a look on this.

**Comment by SUMEKAGARWAL** [comment]
**@khannakshat7 @CodesbyUnnati May I have the confirmation please!**

**Comment by SUMEKAGARWAL** [comment]
**@khannakshat7 @CodesbyUnnati Just have a look on this.**

**Comment by SUMEKAGARWAL** [comment]
**@khannakshat7 @CodesbyUnnati May I have the confirmation please!**

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 @CodesbyUnnati 
Sorry for troubling you. This took some amount of my investment and so I am curious enough to see my PR being merged.
So could you please look into it at least now?...please!

File: electricity/views.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 from django.shortcuts import HttpResponse, redirect, render
 from django.views.decorators.csrf import csrf_exempt
-from django.contrib import messages
-from django.contrib.auth.hashers import make_password, check_password
-from django.contrib.auth.decorators import login_required 
-from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.decorators import login_required
+from django.core.mail import send_mail
+
+from electricity.forms import UserForm
```
Comment (khannakshat7) [inline electricity/views.py:8]: Please pull the latest code from khannakshat7:master and then make a commit

Comment (khannakshat7) [reply electricity/views.py:8]: You are not working on the latest code

Comment (SUMEKAGARWAL) [reply electricity/views.py:8]: Hello bhaiya,@khannakshat7 
Should I change all the commits again or its just for electricity/views,py file?

Comment (khannakshat7) [reply electricity/views.py:8]: Just for views.py file ig

Comment (SUMEKAGARWAL) [reply electricity/views.py:8]: sorry, I am still not clear.
After cloning again the current code, I have to make the changes that I tried in all the files....right?

Comment (khannakshat7) [reply electricity/views.py:8]: So what issue are u facing?

Comment (SUMEKAGARWAL) [reply electricity/views.py]: Please have a look on my new commit once

Comment (SUMEKAGARWAL) [reply electricity/views.py]: Is it fine now?

Comment (SUMEKAGARWAL) [reply electricity/views.py]: Please have a look on my new commit once
Is it fine now?

Comment (SUMEKAGARWAL) [reply electricity/views.py]: Please have a look on this!

**Comment by SUMEKAGARWAL** [comment]
Conflict is resolved!
@khannakshat7

**Review by khannakshat7 — COMMENTED** [review]
Please don't commit settings.py file

**Comment by SUMEKAGARWAL** [comment]
so bhaiya shall I delete from the "Files changed" section?

**Comment by khannakshat7** [comment]
ok try to do that

**Comment by SUMEKAGARWAL** [comment]
Done bhaiya!

**Comment by khannakshat7** [comment]
@SUMEKAGARWAL Sorry I think that was wrong settings.py file got deleted from ur code only. U have to revert the changes of settings.py file and then commit.

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 Bhaiya, I don't know how to revert back the change. Could you please help me through that?

**Comment by SUMEKAGARWAL** [comment]
@khannakshat7 Bhaiya, I have created a new PR starting with #141 .
Shall I close this PR?

**Pull request closed without merging**